data_handler/handlers/helpers.py

```diff
@@ -238,6 +249,16 @@ def get_symbol(address: str) -> str:
     for symbol, addr in symbol_address_map.items():
         if int(addr, base=16) == n:
             return symbol
+
+    # A tuple of that always has this order: `address`, `protocol`.
+    error_info = (address, protocol)
```

Comment:
why you can put it in set in the beginning?

Reply:
should i move it into the `if` statement ?

Reply:
Done